File: 백준/Bronze/2083. 럭비 클럽/럭비 클럽.py
while True:
    name,age,kg = map(str,input().split())
    age = int(age)
    kg = int(kg)
    if name == "#" and age == 0 and kg == 0:
        break
    if age > 17 or kg >= 80:
        print(f"{name} Senior")
    else:
        print(f"{name} Junior")

File: 백준/Bronze/2083. 럭비 클럽/럭비 클럽.c
#include <stdio.h>
#include <stdlib.h>
#include <string.h>
#include <ctype.h>

int main()
{
    while (1) {
        char name[10];
        int age, weight;
        scanf("%s %d %d", &name, &age, &weight);
        if ((age == 0) && (weight == 0)) {
            break;
        }
        if ((age > 17) || (weight >= 80)) {
            printf("%s Senior \n", name);
            continue;
        }
        printf("%s Junior \n", name);
    }

    return 0;
}